pkg/etcd-restore-operator/controller/controller.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 The etcd-operator Authors
```

Comment:
we should remove etcd mentioning in all dir/file names. it is redundant.

Reply:
sounds good.